core/common/src/main/java/alluxio/conf/path/TrieNode.java

```diff
@@ -140,10 +168,56 @@ public TrieNode deleteIf(String path, java.util.function.Function<TrieNode, Bool
   /**
    * @return the iterator of TrieNode that are terminals and have no terminal ancestors
    */
-  public Iterator<TrieNode> getCommonRoots() {
+  public Iterator<TrieNode<V>> getCommonRoots() {
     if (mIsTerminal) {
       return Collections.singletonList(this).iterator();
     }
     return Iterators.concat(mChildren.values().stream().map(TrieNode::getCommonRoots).iterator());
   }
+
+  /**
+   * Get the terminal children of path (including path).
+   * @param path the path
+   * @return the terminal children
+   */
+  public Stream<TrieNode<V>> getChildren(String path) {
```

Comment:
```suggestion
  public Stream<TrieNode<V>> getLeafChildren(String path) {
```
Add `Leaf` or `Terminal` or some other identifier to the name of the function to indicate that it returns only terminal children.